### PR TITLE
fix: IllegalAccessException preventing mono local node run

### LINKED ISSUE #8187: IllegalAccessException preventing mono local node run
I’m getting an exception trying to run the node software (`MonoServicesMain.main(...)`) locally:
```
Caused by: java.lang.IllegalAccessException: symbolic reference class is not accessible: class com.swirlds.platform.uptime.UptimeDataImpl, from class com.swirlds.common.constructable.internal.GenericConstructorRegistry (module com.swirlds.common)
	at java.lang.invoke.MemberName.makeAccessException(MemberName.java:955) ~[?:?]
	at java.lang.invoke.MethodHandles$Lookup.checkSymbolicClass(MethodHandles.java:3686) ~[?:?]
	at java.lang.invoke.MethodHandles$Lookup.resolveOrFail(MethodHandles.java:3646) ~[?:?]
	at java.lang.invoke.MethodHandles$Lookup.findConstructor(MethodHandles.java:2750) ~[?:?]
	at com.swirlds.common.constructable.internal.GenericConstructorRegistry.createConstructorLambda(GenericConstructorRegistry.java:170) ~[swirlds-common-0.42.0-adhoc.xfb77ea8d.jar:?]
	at com.swirlds.common.constructable.internal.GenericConstructorRegistry.registerConstructables(GenericConstructorRegistry.java:105) ~[swirlds-common-0.42.0-adhoc.xfb77ea8d.jar:?]
	at com.swirlds.common.constructable.internal.DefaultConstructableRegistry.registerConstructables(DefaultConstructableRegistry.java:62) ~[swirlds-common-0.42.0-adhoc.xfb77ea8d.jar:?]
	at com.swirlds.platform.util.BootstrapUtils.loadSwirldMains(BootstrapUtils.java:537) ~[swirlds-platform-core-0.42.0-adhoc.xfb77ea8d.jar:?]
	at com.swirlds.platform.Browser.launchUnhandled(Browser.java:202) ~[swirlds-platform-core-0.42.0-adhoc.xfb77ea8d.jar:?]
	at com.swirlds.platform.Browser.launch(Browser.java:170) ~[swirlds-platform-core-0.42.0-adhoc.xfb77ea8d.jar:?]
	... 3 more
```
Branch: `develop`
Swirlds version: `0.42.0-adhoc.xfb77ea8d.jar`

### PR DESCRIPTION
**Pull request by nathanklick**
## Description

This pull request resolves the issues reported by @mhess-swl when running the `MonoServicesMain` class via IntelliJ. 

### Related Issues 

- Closes #8187

### CODE REVIEW

**Review by mhess-swl — APPROVED** [review]
LGTM. Thank you!!

**Review by cody-littley — DISMISSED** [review]
Signing off for the files under hashgraph team jurisdiction.

File: build-logic/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts [hunk 1]
```diff
@@ -50,6 +50,9 @@ extraJavaModuleInfo {
     module("io.grpc:grpc-netty", "grpc.netty") {
         exportAllPackages()
         requireAllDefinedDependencies()
+        dependencies {
+            requires("java.logging")
+        }
```
Comment (jjohannes) [inline build-logic/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts:55]:
```suggestion
        requires("java.logging")
```

Don't need the `dependencies {}` around this.

File: build-logic/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts [hunk 2]
```diff
@@ -70,6 +73,9 @@ extraJavaModuleInfo {
     module("io.grpc:grpc-protobuf-lite", "grpc.protobuf.lite") {
         exportAllPackages()
         requireAllDefinedDependencies()
+        dependencies {
+            requires("com.google.protobuf")
+        }
```
Comment (jjohannes) [inline build-logic/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts:78]:
```suggestion
        requires("com.google.protobuf")
```

**Review by jjohannes — CHANGES_REQUESTED** [review]

**Review by cody-littley — APPROVED** [review]

**Review by jasperpotts — APPROVED** [review]

**Review by kimbor — APPROVED** [review]

**Review by jjohannes — APPROVED** [review]

**Review by jeromy-cannon — APPROVED** [review]

**Pull request merged**